src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -214,9 +241,14 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
                     )
                     descriptionPanel.add(scrollPane, BorderLayout.CENTER)
 
-                    service<SnykAnalyticsService>().logEvent(
-                        Segment.Event.USER_SEES_AN_ISSUE,
-                        EventPropertiesProvider.getIssueDetailsForOpenSource(groupedVulns)
+                    val issue = groupedVulns.first()
+                    service<SnykAnalyticsService>().logIssueIsViewed(
+                        IssueIsViewed.builder()
+                            .ide(IssueIsViewed.Ide.JETBRAINS)
+                            .issueId(issue.id)
+                            .issueType(issue.getIssueType())
+                            .severity(issue.getIssueSeverityOrNull())
```

Comment:
will it accept Null?

Reply:
yes, because it's object (but itly validation will fail).